ultet/examples/database/src/main/my_schema/my_table.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2023 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+table: my_schema.my_table
+description: This is an example table
+primary_db: example_db
+owner: some_owner_user
+columns:
+- id_key_field:
+    type: bigint
```

Comment:
Does you expect to have comment on this line? Maybe they would be usefull.

Reply:
Comment on what? The field comment is in the description field.